pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -260,6 +270,55 @@ func TestLocalTaskRef(t *testing.T) {
 	}
 }
 
+func TestLocalTaskRef_Error(t *testing.T) {
```

Comment:
Thanks for the test case @JeromeJu! The code coverage has increase.

I think it makes more sense to refactor the `TestLocalTaskRef_Error` test function with the above TestLocalTaskRef() as they are testing the case function, and most of them seems duplicate. You can change the `wantErr` from bool to the real error type; and putting the namespace to each test case？

Reply:
Thanks @QuanZhang-William , updated:)